src/lib/serviceRoutinesV2/getEntities.cpp

```diff
@@ -61,17 +60,58 @@ std::string getEntities
   ParseData*                 parseDataP
 )
 {
-  std::string  answer;
-  Entities     entities;
+  using namespace std;
+  typedef vector<string>  vstr;
+  typedef vstr::size_type vstr_sz;
+
+  Entities entities;
+  string   answer;
+
+  // optional parameter for list of IDs
+  string paramIDs     = ciP->uriParam["id"];
+  string paramPattern = ciP->uriParam["idPattern"];
+  string pattern      = ".*"; // all entities, default value
 
+  if (paramPattern != "" && paramIDs != "")
+  {
+    ciP->httpStatusCode = SccBadRequest;
+    answer = "{\"error\":\"Incompatible parameters: id, IdPattern\"}";
```

Comment:
details put in error field.
It should be something like:

    {
        "error": "BadRequest",
        "description": "Incompatible parameters: id, IdPattern"
    }

However, we have classes and macros for this.
In my opinion (I know it gets slower in run-time, but things get a little bit ordered, at least), OrionError should be used.